### DISCUSSION TITLE
04273FC4-05BC-1936-BC09-F502E07D7696

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#106] [General]
# 04273FC4-05BC-1936-BC09-F502E07D7696

Kepler-24 is a Sun-like star about 3,900 light-years away in the constellation Lyra. Four confirmed planets (three Neptunians and one superterran) are found orbiting very close to their parent star

https://celestia.mobi/addon?item=04273FC4-05BC-1936-BC09-F502E07D7696

**Comment by TheLostProbe** [comment]
there's a visible seam in the image 💀💀

**Reply by Karistus** [reply]
Not anymore

**Reply by TheLostProbe** [reply]
you are doing god's work, thank you 🙏🙏
